_static/AuxTelCalIll_FunctionalTest.ipynb

```diff
@@ -220,6 +241,32 @@
     "tmp = await WhiteLightSource.cmd_turnLampOn.set_start(power=1000)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7c6c45bf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# When you are done with all testing, turn off the White Light Source, \n",
+    "tmp = await WhiteLightSource.cmd_turnLampOff.set_start()\n",
+    "## note, you can also send it a Force=True if there is some issue and you need to turn off the lamp\n",
+    "\n",
+    "# Wait 10 minutes, then turn off the Chiller\n",
```

Comment:
You kept this stopChiller command in the same as the turnLampOff.
So if you tell the chiller to stop, even at the same time.. it'll continue operating for 10 minutes and then turn off, correct?
If so, maybe clarify that in the text otherwise it's a bit confusing as you might be telling the user to wait 10 minutes before commanding the chiller off.